script/approve_and_execute.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# Imperfect but simple script to pass governance proposals on Celo tesnets (staging, baklava or alfajores)
+set -euo pipefail
+
+# Variables to be defined
+# PROPOSAL_JSON_PATH= # Path for json of the proposal
+# DESCRIPTION_URL= # Path for the URL containing the proposal
+SIGNER=0xfCf982bb4015852e706100B14E21f947a5Bb718E # address that will send the tx
+# APPROVER=0xfCf982bb4015852e706100B14E21f947a5Bb718E # approver address
```

Comment:
This script needs some work to be more generalized. I wouldn't just remove things from it because that makes it harder to use. For example, keeping that `PROPOSAL_ID=74` line, makes it evident that you need to put the value of the proposal you want to execute there. If you remove it, you'll have to waste time to realize that you need to add it.

Same for the commented-out variables. All are necessary but I commented out some, because I was loading them from a local `.env` file for security reasons.

We should give this script more love the next time we deploy to Baklava, after the audit fixes.

Reply:
ok fair!